docs/developer/extending/apps/developing-apps/app-sdk/apl.md

```diff
@@ -71,19 +71,29 @@ import { APL, AuthData } from "@saleor/app-sdk/apl";
 const client = createClient();
 await client.connect();
 
+// If you intend to use the same Redis instance for multiple Apps,
+// add prefix to the keys you will keep AuthData:
+// - `APPID1:http://staging.saleor.io/graphql`
+// - `APPID1:http://demo.saleor.io/graphql`
+// - `APPID2:http://demo.saleor.io/graphql`
+const prepareKey = (apiUrl: string) => `${APP_ID}:${apiUrl}`;
```

Comment:
**polish**: I know the function isn't too complicated, but I suggest using a more descriptive name.